Add print newline, make WARNING into INFO and refactor

pgdr opened the pull request:
4 minor changes:

1. add a newline terminator in two print statements
2. make a warning into a message ("active information loaded from...")
3. add some more detail in a _warning_ to better debug the issue
4. fix a const and remove an unused static function in `ext_job`

joakim-hove commented on libenkf/src/enkf_state.c:
```diff
@@ -528,7 +528,7 @@ static void enkf_state_internalize_GEN_DATA(const ensemble_config_type * ens_con
   if (stringlist_get_size( keylist_GEN_DATA) > 0)
     if (last_report <= 0)
-      res_log_warning("Trying to load GEN_DATA without properly "
+      res_log_info("Trying to load GEN_DATA without properly "
```
This is my opinion `WARNING`.

joakim-hove commented on libenkf/src/gen_data.c:
```diff
@@ -238,50 +238,53 @@ static void gen_data_set_data__(gen_data_type * gen_data , int size, const forwa
+                 __func__, active_file);
+  }
+  fclose(stream);
+  res_log_finfo("GEN_DATA(%s): active information loaded from:%s.",
```
This is fine as `INFO`!

Commit:
Add newline in print statement and minor fixups

Make a warning into an info
Add more information in a warning for debugging the warning
Fix a missing const in a static function in ext_job
